README.md

```diff
@@ -101,6 +104,30 @@ class Aircraft(models.Model):
 | `QuerySet.update()`           | No
 | `QuerySet.update_or_create()` | Yes (via `QuerySet.get_or_create()` and `Model.save()`)
 
+#### Auditing Special QuerySet Writes
+
+Auditing for the four "special" QuerySet methods that perform DB writes (labeled
+**No** in the table above) _can_ be enabled. This requires three extra usage
+details:
+
+1. Enable the feature by calling the audit decorator specifying
+   `@audit_fields(..., audit_special_queryset_writes=True)`.
```

Comment:
Wondering if this should just be the default behavior. Can you share your thoughts on that?

Reply:
It's not the default due to the extra workflow required (and performance implications that come along) when you enable it. Making this the default would require all "customers" to perform extra boilerplate just to use the simplest workflow.